workbench-core/example/app/src/routes/dynamicAuthorizationRoutes.ts

```diff
@@ -96,4 +100,27 @@ export function setUpDynamicAuthorizationRoutes(router: Router, service: Dynamic
       }
     })
   );
+
+  router.get(
+    '/authorization/groups/:groupId/getUsers',
+    wrapAsync(async (req: Request, res: Response) => {
+      try {
+        const validatedRequest = validateAndParse<GetUsersFromGroupRequest>(
+          GetUsersFromGroupRequestParser,
+          req.params
+        );
```

Comment:
ah, actually since its parsing the params, we dont need this. It wont make it to this route if the request params arent correct

Reply:
isn't that the case for https://github.com/aws-solutions/solution-spark-on-aws/blob/GetGroupUsers/workbench-core/example/app/src/routes/dynamicAuthorizationRoutes.ts#L59-L65 ?

Reply:
Updated in #730

Reply:
Fixed !